packages/address/src/__tests__/Utils.spec.ts

```diff
@@ -22,7 +22,7 @@ describe('bs58ToBin', () => {
     const address = new Address(0, NetTypes.MAINNET, 1, bob.publicKey).b58
     const bin = bs58.decode(address)
     const vPayload = bin.slice(0, -4)
-    const checksum = bin.slice(-4)
+    const checksum = Buffer.from(bin.slice(-4))
```

Comment:
Why was this change needed?

Reply:
This was from me. Tests were failing without this, not sure if this was environment specific though.

Reply:
I'll see if I can remove it

Reply:
@tyler-whitman - Looks required to me, this test fails without the update.